localization/english/culture/wc_culture_titles_l_english.yml

```diff
@@ -1,43 +1,390 @@
+ chieftess_female_old_dwarven_count:0 "Chieftess"
+ chiefdom_old_dwarven_county:0 "Chiefdom"
+ thane_male_old_dwarven_duke:0 "Thane"
+ thane_female_old_dwarven_duke:0 "Thane"
+ high_chiefdom_old_dwarven_king:0 "High Chiefdom"
+ high_thane_male_old_dwarven_king:0 "High Thane"
+ high_thane_female_old_dwarven_kingdom:0 "High Thane"
+
+ ### Vrykul and Kvaldir ###
+ # Feudal
+ thane_male_vrykul_count:0 "Thane"
+ thane_female_vrykul_count:0 "Thane"
+ chiefdom_vrykul_county:0 "Chiefdom"
+ jarl_male_vrykul_duke:0 "Jarl"
+ frue_female_vrykul_duke:0 "Frue"
+ jarldom_vrykul_duchy:0 "Jarldom"
+ high_king_male_vrykul_emperor:0 "High King"
+ high_queen_female_vrykul_emperor:0 "High Queen"
+ high_kingdom_vrykul_empire:0 "High Kingdom"
+
+ ### Frost Vrykul ###*
+ # Feudal
+ thane_male_frost_vrykul_count:0 "Thane"
+ thane_female_frost_vrykul_count:0 "Thane"
+ chiefdom_frost_vrykul_county:0 "Chiefdom"
+ overthane_male_frost_vrykul_duke:0 "Overthane"
+ overthane_female_frost_vrykul_duke:0 "Overthane"
+ jarldom_frost_vrykul_duchy:0 "Jarldom"
+ high_king_male_frost_vrykul_emperor:0 "High King"
+ high_queen_female_frost_vrykul_emperor:0 "High Queen"
+ high_kingdom_frost_vrykul_empire:0 "High Kingdom"
+
+ ### Staroveric ###
+ # Feudal
+ grand_prince_male_solstice_duke:0 "Grand Prince"
+ grand_princess_female_solstice_duke:0 "Grand Princess"
+ grand_principality_solstice_duchy:0 "Grand Principality"
+ tsar_male_solstice_emperor:0 "Tsar"
+ tsaritsa_female_solstice_emperor:0 "Tsaritsa"
+ tsardom_solstice_empire:0 "Tsardom"
+
+ ### Dryad group ###
+ # Feudal / Republic / Clan / Theocratic / Tribal
+ tender_male_dryad_group_baron:0 "Tender"
+ tender_female_dryad_group_baron:0 "Tender"
+ refuge_dryad_group_baronny:0 "Refuge"
+ guardian_male_dryad_group_count:0 "Guardian"
+ guardian_female_dryad_group_count:0 "Guardian"
+ grove_dryad_group_county:0 "Grove"
+ lord_male_dryad_group_duke:0 "Lord"
+ lady_female_dryad_group_duke:0 "Lady"
+ grand_grove_dryad_group_duchy:0 "Grand Grove"
+ high_lord_male_dryad_group_king:0 "High Lord"
+ high_lady_female_dryad_group_king:0 "High Lady"
+ domain_dryad_group_kingdom:0 "Domain"
+ king_male_dryad_group_emperor:0 "King"
+ queen_female_dryad_group_emperor:0 "Queen"
+ kingdom_dryad_group_empire:0 "Kingdom"
+
+ ### Tol'vir ###
+ # Feudal - Normal
+ hatya_male_tolvir_baron:0 "Hatya"
+ hatya_female_tolvir_baron:0 "Hatya"
+ niout_tolvir_barony:0 "Niout"
+ heqa_male_tolvir_count:0 "Heqa"
+ heqa_female_tolvir_count:0 "Heqa"
+ hout_tolvir_county:0 "Hout"
+ nomarch_male_tolvir_duke:0 "Nomarch"
+ nomarch_female_tolvir_duke:0 "Nomarch"
+ nome_tolvir_duchy:0 "Nome"
+ pharaoh_male_tolvir_king:0 "Pharaoh"
+ pharaoh_female_tolvir_king:0 "Pharaoh"
+
+ # Feudal - Evil religions
+ dark_pharaoh_male_tolvir_king:0 "Dark Pharaoh"
+ dark_pharaoh_female_tolvir_king:0 "Dark Pharaoh"
+
+ vizier_male_tolvir:0 "Vizier"
+ vizier_female_tolvir:0 "Vizier"
+ grand_vizier_male_tolvir:0 "Grand Vizier"
+ grand_vizier_female_tolvir:0 "Grand Vizier"
```

Comment:
It's weird councilors of dukes, counts aren't named viziers and have generic name.

Reply:
The viziers are for royal titles. The lower titles never had viziers. In Egypt.

Reply:
Do you want to localize this as well?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/135925091-419752ae-84ef-466a-b435-f99df9aefeca.png)

</details>

Reply:
what do you mean ? "Your Chancellor" ? If yes, it's not really flavorization so it would have to be done in another PR.

Reply:
> what do you mean ? "Your Chancellor" ? If yes, it's not really flavorization so it would have to be done in another PR.

Yes. Open a new issue then or move councilor titles to another branch.

Reply:
It working, I don't understand what do you mean

Reply:
> It working, I don't understand what do you mean

The title is called Chancellor in the GUI. I expected it to be fully localized.

Reply:
> Ah ok, but that's not flavorization, it's another localization. I only touched the flavorization folder.

Mmmkay, you may move to another branch if you don't want to change other files here.

Reply:
Well, we're not going to discuss for 15 years either. Deleted in d706d80d9